Corrected Spelling (#131)
qmk/qmk_toolbox: readme.md

### Download

> A Windows and macOS version is available, and you can get [the latest release here](https://github.com/qmk/qmk_toolbox/releases).

Reviewer: I think this is a more accurate phrasing:

```suggestion
Windows and macOS versions are available, and you can get [the latest release here](https://github.com/qmk/qmk_toolbox/releases).
```
